lib/src/chart/line_chart/line_chart_painter.dart

```diff
@@ -37,7 +39,7 @@ class LineChartPainter extends AxisChartPainter<LineChartData> with TouchHandler
     {double textScale})
     : super(data, targetData, textScale: textScale) {
     touchHandler(this);
-    
+
```

Comment:
And please prevent to left these changes.

Reply:
my editor really hates this for some reason and will not let me even without a formatter turned on. i will need to manually add these spaces back in

Reply:
can we just leave this in? even if I add these spaces back most any editor will remove them again anytime this project is worked on. there's no reason for random spaces to be on an empty line

Reply:
I'm totally with you (this should not be there) but the point is that we should handle a merge request only for a single feature, or improvement,
we can have a separated PR that all of dart files are formatted and I will merge it immediately.
But it's okay for this, left it.